### ISSUE TITLE
XDebug not formatting var_dump after updating vvv to 3.5.1

### ISSUE THREAD

**Issue by timichango** [Varying-Vagrant-Vagrants/VVV#2343]
I updated to VVV 3.5.1 earlier today, and noticed that output from `var_dump()` is now not formatted. 

Browsing recent documentation, I'm now aware that xdebug itself was updated to v3, and accordingly, I've tried using the new command `switch_php_debugmod xdebug` to enable debugging — which successfully activates xdebug, seemingly, but yet xdebug is _**still**_ not formatting `var_dump()` output.

Anyone have any ideas about how to get this working properly again?

Thank you!

**Comment by tomjn** [comment]
Did you try this on the `develop` branch? This is a known/fixed issue, Xdebug 3 did not have development aids turned on.

I'm going to close this as already fixed in `3.6` via https://github.com/Varying-Vagrant-Vagrants/VVV/commit/f37068217f418b260f30322e5f1a54a539ea5688, if this  is still an issue on the `develop` branch then we can reopen

**Comment by timichango** [comment]
Hey @tomjn — thanks! Was actually just coming back here right now to close this out myself, as I managed to resolve it (figured out that it was tied to the new xdebug modes, and dev aids being disabled, as you mentioned). 

I didn't try develop, nor was I even aware that was possible — I simply updated VVV using `git pull`, per the documentation, when prompted during an unrelated earlier re-provision, which is what brought me up to the latest release, 3.5.1, and when the problem arose.

**For anyone else in the same position, and who wants a quick fix without switching to the develop branch:** in `config/php-config/xdebug.ini` just changing the line `xdebug.mode=debug` to `xdebug.mode=develop`, then re-provisioning and starting xdebug seems to do the trick.
